--- onlinel2filter.py.orig	2020-02-23 17:00:58.204752000 +0000
+++ onlinel2filter.py	2020-02-23 17:01:17.340752000 +0000
@@ -403,17 +403,6 @@
     ##########################
 
 
-    #############################################
-    # Store pulse mask for pulses used by recos #
-    #############################################
-
-    tray.AddModule(PulseMaskShortcutter, name + "_PulseMaskShortcutter",
-            PulseMaskName  = pulses,
-            ShortcutName   = name+"_"+pulses,
-            OrigSourceName = "I3SuperDST",
-            If = lambda f: If(f) and DoAdvancedReco(f) )
-
-
     #################################
     # Cramer-Rao for BestFit so far #
     #################################
